README.md

```diff
@@ -1,4 +1,35 @@
-# Safe Apps SDK
+# Gnosis Safe apps SDK
+
+[![Logo](https://raw.githubusercontent.com/gnosis/safe-apps-sdk/master/assets/logo.png)](https://gnosis.pm/)
+
+[![Build Status](https://travis-ci.org/gnosis/safe-apps-sdk.svg?branch=master)](https://travis-ci.org/gnosis/pm-contracts)
+
+Software development kit to integrate a third-party app with Safe Multisig (https://gnosis-safe.io/app/).
 
 ## Install
-- `yarn`
+
+### Install requirements with yarn:
+
+```bash
+yarn install @gnosis.pm/safe-app-sdk
+```
+
+## Build
+
+```bash
```

Comment:
don't you have to run `yarn` before to install the dependencies?

Reply:
yarn or yarn install it's the same I think.

Reply:
yes, but right now it doesn't say either, right? :P ... again most devs should be able to figure that out :D

Reply:
I'll add it anyways, but it only for lib maintainers, final users only need to install and import it.